3.3/examples/django/bootstrap.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+OPENSHIFT_LOGFILE=openshift.log
+OPENSHIFT_PIDFILE=openshift.pid
+
+if [ -f $OPENSHIFT_PIDFILE ]; then
+	echo "OpenShift is already running, run ./cleanup.sh and try again"
+	exit 1
+fi
+
+if [ $USER = vagrant ]; then
+	OPENSHIFT_BIN=/data/src/github.com/openshift/origin/_output/local/go/bin/openshift 
+	OPENSHIT_PARAMS="--public-master=localhost"
+else
+	OPENSHIFT_BIN=openshift
+fi
+
+# Start OpenShift in the background
+echo "-> starting OpenShift in the background...  (tail -f $OPENSHIFT_LOGFILE)"
```

Comment:
I'll remove this script and point to the origin/sample-app for instructions in the README.md, keeping those steps just for myself.